tensorflow/contrib/learn/python/learn/estimators/estimator.py

```diff
@@ -343,7 +348,7 @@ def config(self):
       SCIKIT_DECOUPLE_DATE, SCIKIT_DECOUPLE_INSTRUCTIONS, 'x', 'y', 'batch_size'
   )
   def fit(self, x=None, y=None, input_fn=None, steps=None, batch_size=None,
-          monitors=None, max_steps=None):
+          monitors=None, max_steps=None, *input_fn_args, **input_fn_kwargs):
```

Comment:
I suppose, although it would be nice for users to be able to specify require arguments without having to resort to:
```
if arg is None:
  raise ValueError()
```
or a downstream error

Still I'm sensitive to the backwards compat issues taking arbitrary positional args raises. I still prefer allowing a catchall pass through here, but if you feel strongly I can remove it (or replace it with:
```
def fit(..., input_fn_args=[], **input_fn_kwargs)
```
even.